crates/task-impls/src/vote_collection.rs

```diff
@@ -272,6 +272,7 @@ impl<TYPES: NodeType> AggregatableVote<TYPES, UpgradeVote<TYPES>, UpgradeCertifi
         certificate: UpgradeCertificate<TYPES>,
         _key: &TYPES::SignatureKey,
     ) -> HotShotEvent<TYPES> {
+        tracing::error!("Upgrade certificate formed!");
```

Comment:
Sorry if I already commented about this, is this a debugging message left behind? Or do we want to notify this as an error?

Reply:
Oops, yes -- another leftover. Thanks for catching it -- removed!